recipes/fasttext/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fasttext" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8c4ba14372d160fe7de902299caac664b7c1af1778601e17ca18ca2081b0e253" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 431c9e2.tar.gz
```

Comment:
Please change to `{{ name }}-{{ version }}.tar.gz` – this key is confusing, it doesn't refer to the last component of the url but to the local filename that conda will keep in the cache.